src/connector/src/connector_common/mqtt_common.rs

```diff
@@ -141,26 +142,22 @@ impl MqttCommon {
             .unwrap_or(QoS::AtMostOnce)
     }
 
-    fn get_tls_config(&self) -> ConnectorResult<tokio_rustls::rustls::ClientConfig> {
-        let mut root_cert_store = tokio_rustls::rustls::RootCertStore::empty();
+    fn get_tls_config(&self) -> ConnectorResult<rustls::ClientConfig> {
+        let mut root_cert_store = rustls::RootCertStore::empty();
         if let Some(ca) = &self.ca {
             let certificates = load_certs(ca)?;
             for cert in certificates {
-                root_cert_store.add(&cert).unwrap();
+                root_cert_store.add(cert).unwrap();
             }
         } else {
             for cert in
                 rustls_native_certs::load_native_certs().expect("could not load platform certs")
             {
-                root_cert_store
-                    .add(&tokio_rustls::rustls::Certificate(cert.to_vec()))
-                    .unwrap();
+                root_cert_store.add(cert).unwrap();
             }
         }
 
-        let builder = tokio_rustls::rustls::ClientConfig::builder()
-            .with_safe_defaults()
-            .with_root_certificates(root_cert_store);
+        let builder = rustls::ClientConfig::builder().with_root_certificates(root_cert_store);
```

Comment:
https://github.com/rustls/rustls/releases/tag/v%2F0.22.0
> `ConfigBuilder::with_safe_defaults` - calls to this can simply be deleted since safe defaults are now implicit.